website/client/components/shared/inventoryDrawer.vue

```diff
@@ -0,0 +1,169 @@
+<template lang="pug">
+drawer.inventoryDrawer(
+  :title="$t('quickInventory')"
+  :errorMessage="inventoryDrawerErrorMessage(selectedDrawerItemType)"
+)
+  div(slot="drawer-header")
+    drawer-header-tabs(
+      :tabs="filteredTabs",
+      @changedPosition="tabSelected($event)"
+    )
+      div(slot="right-item")
+        #petLikeToEatMarket.drawer-help-text(v-once)
+          | {{ $t('petLikeToEat') + ' ' }}
+          span.svg-icon.inline.icon-16(v-html="icons.information")
+        b-popover(
+          target="petLikeToEatMarket",
+          :placement="'top'",
+        )
+          .popover-content-text(v-html="$t('petLikeToEatText')", v-once)
+
+  drawer-slider(
+    v-if="hasOwnedItemsForType(selectedDrawerItemType)"
+    :items="ownedItems(selectedDrawerItemType) || []",
+    slot="drawer-slider",
+    :itemWidth=94,
+    :itemMargin=24,
+    :itemType="selectedDrawerTab"
+  )
+    template(slot="item", slot-scope="ctx")
+      slot(
+        name="item",
+        :item="ctx.item",
+        :itemClass="getItemClass(selectedDrawerContentType, ctx.item.key)",
+        :itemCount="userItems[selectedDrawerContentType][ctx.item.key] || 0",
+        :itemName="getItemName(selectedDrawerItemType, ctx.item)",
+        :itemType="selectedDrawerItemType"
+      )
+
+
+</template>
+
+<script>
+  import {mapState} from 'client/libs/store';
+  import inventoryUtils from 'client/mixins/inventoryUtils';
+  import svgInformation from 'assets/svg/information.svg';
+  import _filter from 'lodash/filter';
+
+  import CountBadge from 'client/components/ui/countBadge';
+  import Item from 'client/components/inventory/item';
+  import Drawer from 'client/components/ui/drawer';
+  import DrawerSlider from 'client/components/ui/drawerSlider';
+  import DrawerHeaderTabs from 'client/components/ui/drawerHeaderTabs';
+
+  export default {
+    mixins: [inventoryUtils],
+    components: {
+      Item,
+      CountBadge,
+      Drawer,
+      DrawerSlider,
+      DrawerHeaderTabs,
+    },
+    props: {
+      defaultSelectedTab: {
+        type: Number,
+        default: 0,
+      },
+      showEggs: Boolean,
+      showPotions: Boolean,
+    },
+    data () {
+      return {
+        drawerTabs: [
+          {
+            key: 'eggs',
+            label: this.$t('eggs'),
+            show: () => this.showEggs,
+          },
+          {
+            key: 'food',
+            label: this.$t('foodTitle'),
+            show: () => true,
+          },
+          {
+            key: 'hatchingPotions',
+            label: this.$t('hatchingPotions'),
+            show: () => this.showPotions,
+          },
+          {
+            key: 'special',
+            contentType: 'food',
+            label: this.$t('special'),
+            show: () => true,
+          },
+        ],
+        selectedDrawerTab: this.defaultSelectedTab,
```

Comment:
does it work to reference a prop here ?

Reply:
yep :) at first I thought I'd have to use something like `watch` for that but it worked just like that